server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts

```diff
@@ -580,11 +581,19 @@ export class SummaryWriter implements ISummaryWriter {
 	}
 
 	private async generateLogtailEntries(
-		from: number,
-		to: number,
+		gt: number,
+		lt: number,
 		pending: ISequencedOperationMessage[],
 		lastSummaryMessages: ISequencedDocumentMessage[] | undefined,
 	): Promise<ITreeEntry[]> {
+		let to = lt;
+		const from = gt;
```

Comment:
@pradeepvairamani

We have seen `gt` `lt` are sometimes null/undefined. Could we fix that as well?

Reply:
Hey Xin..It is very unlikely that gt and lt are undefined in this flow. That might be happening as a part of the lambda factory creation. https://github.com/microsoft/FluidFramework/blob/7b8e75cfe633bba05e78ca4979df3e6a6dbe4656/server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts#L302. I will reach out to you offline to discuss this scenario with you, but I am inclined to keep that out of the scope of this PR since this is limited to restricting the logtail size in summaries.